crates/ekore/src/harmonics/cache.rs

```diff
@@ -46,6 +46,12 @@ pub enum K {
     Sm21e,
     /// $S_{-2,1}(N)$ odd moments
     Sm21o,
+    /// recursive harmonics
+    S1ph,
+    S2ph,
+    S3ph,
+    S1p2,
+    G3p2,
```

Comment:
I'm not sure we want to keep those - can we inline them?

Reply:
what do you mean? Could you please provide an example?

Reply:
instead of
```py
    let g3N = c.get(K::G3);
    let g3Np2 = c.get(K::G3p2);
```
we do
```py
    let g3N = c.get(K::G3);
    let g3Np2 = g3N + f;
```
where $f = g_3(N+2)-g_3(N)$ is computed analytically beforehand

Reply:
I am afraid I don t understand. How do you compute f analytically beforehand? Do you mean to move `g_functions::g3(self.n + 2., self.get(K::S1p2))` out of the cache?

Reply:
> out of the cache?

yes

> How do you compute f analytically beforehand?

Mathematica :upside_down_face: let's try to use a short cut: @giacomomagni can you compute quickly this $f$? (also feel free to give your opinion if we should keep that cache entry or not)

Reply:
so I think
$$f = \frac{6 (n+1) (2 n+1) H_n+n \left(-\pi ^2 (n+1)^2-6 n\right)}{6 n^2 (n+1)^3}$$
which I computed half with pen and paper and half Mathematica and checked for $N=1$, $N=2$

to be more precise: for $g_3$ we could use the above equation and for the others we inline the expressions

Reply:
ok thank you! I ll first try to get things working as they are and then I will add this